### PR TITLE
Default quarkus.kafka-streams.application-id to quarkus.application.name

### PR DESCRIPTION
**Pull request by codeJack**
As part of issue #[3500](https://github.com/quarkusio/quarkus/issues/3500) : Default quarkus.kafka-streams.application-id to quarkus.application.name.

Purpose of this PR is to simplify configuration for the kafka-streams extension by providing a default value for the [application-id](https://quarkus.io/guides/all-config#quarkus-kafka-streams_quarkus.kafka-streams.application-id) parameter that would be quarkus' [application.name](https://quarkus.io/guides/all-config#quarkus-core_quarkus.application.name).

### CODE REVIEW

**Comment by geoand** [comment]
The CI issue has nothing to do with this PR:

```
Unable to find image 'quay.io/keycloak/keycloak:10.0.0' locally
docker: Error response from daemon: received unexpected HTTP status: 500 Internal Server Error.
See 'docker run --help'.
##[error]Process completed with exit code 125.
```

**Comment by geoand** [comment]
Thanks for this!

It makes sense to, @gunnarmorling WDYT?

File: docs/src/main/asciidoc/kafka-streams.adoc [hunk 1]
```diff
@@ -469,7 +468,7 @@ kafka-streams.metrics.recording.level=DEBUG
 
 The options with the `quarkus.kafka-streams` prefix can be changed dynamically at application startup,
 e.g. via environment variables or system properties.
-`bootstrap-servers`, `application-id` and `application-server` are mapped to the Kafka Streams properties `bootstrap.servers`, `application.id` and `application.server`, respectively.
+`bootstrap-servers` and `application-server` are mapped to the Kafka Streams properties `bootstrap.servers`, `application.id` and `application.server`, respectively.
```
Comment (gunnarmorling) [inline docs/src/main/asciidoc/kafka-streams.adoc:471]:
```suggestion
`bootstrap-servers` and `application-server` are mapped to the Kafka Streams properties `bootstrap.servers` and `application.server`, respectively.
```

**Review by gunnarmorling — APPROVED** [review]
Yes, good idea! One small doc fix suggested inline.

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
LGTM, approved.

**Review by geoand — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**